WEB/Src/WindowsServer/WindowsServer/Implementation/HeartbeatPropertyManagerProvider.cs

```diff
@@ -14,7 +14,7 @@ public static IHeartbeatPropertyManager GetHeartbeatPropertyManager()
             {
                 try
                 {
-                    foreach (var module in telemetryModules.Modules)
```

Comment:
qn - why do we still access .Modules here?

Reply:
this is a helper class for net full scenarios. 
looks like I changed the whitespace here. I'll remove it.